tailwind.config.js

```diff
@@ -128,7 +130,8 @@ module.exports = {
         "3xl": "1px 1px 14px rgba(0, 0, 0, 0.1)",
       },
       transitionDuration: {
-        '0.4': '0.4s',
+        0.2: '0.2s',
```

Comment:
You should use the recommended tailwindcss way of using 200 or 400 instead of 0.2 and 0.4

Reply:
yeah sure let me go with that.